src/Interactivity/directives/class-woo-directive-store.php

```diff
@@ -11,10 +11,6 @@ static function merge_data( $data ) {
 		self::$store = array_replace_recursive( self::$store, $data );
 	}
 
-	static function serialize() {
-		return json_encode( self::$store );
-	}
-
```

Comment:
I guess the one thing to check would be that this method wasn't called externally (though I'm pretty sure it wasn't).